### PR TITLE
Add Actions for Archiving Dependencies

### PR DESCRIPTION
**Pull request by dragonstyle**
Deno, Pandoc, and TinyTex.

Depends upon setting up Buckets, which is not yet complete.

### CODE REVIEW

**Comment by jjallaire** [comment]
LGTM, merge when ready!

**Review by jjallaire — APPROVED** [review]

**Comment by jjallaire** [comment]
We should add pandoc-crossref here as well

File: .github/tinytex.yml [hunk 1]
```diff
@@ -0,0 +1,54 @@
...
+     runs-on: ubuntu-latest
+
+     # Steps represent a sequence of tasks that will be executed as part of the job
+     steps:
+     # Runs a single command using the runners shell
+     - name: Download Local Copies
+       run: |
+            for EXT in $EXTS; do
+             FILENAME=TinyTeX-${{ github.event.inputs.version }}.$EXT
+             curl -fail -L -o $FILENAME https://github.com/yihui/tinytex-releases/releases/download/${{ github.event.inputs.version }}/$FILENAME
```
Comment (jjallaire) [inline .github/tinytex.yml:36]: Which variation are we targeting? (looks like there are 3: https://github.com/yihui/tinytex-releases#releases)

Comment (dragonstyle) [reply .github/tinytex.yml:36]: I targeted the `TinyTeX.*` with this, but looking merely closely, I think we could get away with `TinyTeX-1.*` which would be enough to compile most RMarkdown documents. What do you think?

**Pull request merged**